tutorial.ipynb

```diff
@@ -0,0 +1,665 @@
+      "text/plain": [
+       "<Figure size 432x288 with 3 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "subj_hit_trials = hit_trials[0]\n",
+    "subj_miss_trials = [idx for idx in range(n_trials) if idx not in subj_hit_trials]\n",
+    "vlim = [0, 6.0e-17]\n",
+    "\n",
+    "fig = se.plot_avg_spectrogram(tfr=tfrs[subj_hit_trials],\n",
+    "                              times=times,\n",
+    "                              freqs=freqs,\n",
+    "                              event_band=event_band,\n",
+    "                              vlim=vlim)  # note the vlim arguement!\n",
+    "\n",
+    "fig = se.plot_avg_spectrogram(tfr=tfrs[subj_miss_trials],\n",
+    "                              times=times,\n",
+    "                              freqs=freqs,\n",
+    "                              event_band=event_band,\n",
+    "                              vlim=vlim)  # note the vlim arguement!"
```

Comment:
You should say what `vlim` is in units somewhere if you're drawing attention to it

Reply:
Besides saying "spectral power", It actually depends on the units of the timeseries data that the user reads in.